code/modules/client/preference_setup/loadout/loadout_fluffitems_yw.dm

```diff
@@ -681,6 +690,24 @@
 	ckeywhitelist = list("masmc")
 	character_name = list("Kettek Ollarch")
 
+//Moca_The_Porg1
+//Mocha
+/datum/gear/fluff/mochaexploreruniform
+	path = /obj/item/clothing/under/fluff/mocha_uniform
+	display_name = "Winterized Explorer Jumpsuit"
+	description = "A Greenish-white uniform for operating in hazardous environments. This one is suited for colder environments."
+	ckeywhitelist = list("Moca_The_Porg1")
+	character_name = list("Mocha")
+	allowed_roles = list("Explorer", "Pathfinder")
+
+/datum/gear/fluff/mocha_suit_kit
+	path = /obj/item/clothing/suit/storage/hooded/fluff/mocha_suit
```

Comment:
That is still the path of the suit itself and not the modkit. Since you already added the 'allowed roles' thing you can probably delete the modkit, or just fix the path to point to the modkit.